Docs Update: Add Reown as one of the Dev Tools available on Rootstock (#186)
rsksmart/devportal: docs/05-dev-tools/user-onboarding/reown.md

---
sidebar_position: 101
sidebar_label: Reown
title: User Onboarding with Reown
description: "Reown gives developers the tools to build user experiences that make digital ownership effortless, intuitive, and secure." 

> tags: [reown, account abstraction, developer tools, rootstock, ethereum, dApps, wallets, solana, wallet connections, wallet infrastructure]

Reviewer: ```suggestion
tags: [reown, account abstraction, developer tools, rootstock, ethereum, dApps, wallets, wallet connections, wallet infrastructure]
```

---

> # Reown (prev. known as WalletConnect)

Reviewer: ```suggestion
```

> **[Reown](https://reown.com/?utm_source=rootstock&utm_medium=docs&utm_campaign=backlinks)** gives developers the tools to build user experiences that make digital ownership effortless, intuitive, and secure.

Reviewer: ```suggestion
**[Reown](https://reown.com/?utm_source=rootstock&utm_medium=docs&utm_campaign=backlinks)** provides developers with the tools to build user experiences that make digital ownership effortless, intuitive, and secure.
```

Reown has two major product offerings, they are, **AppKit** and **WalletKit**.

> ### AppKit

Reviewer: ```suggestion
## AppKit
```

AppKit is a powerful, free, and fully open-source SDK for developers looking to integrate wallet connections and other Web3 functionalities into their apps on any EVM and non-EVM chain. In just a few simple steps, you can provide your users with seamless wallet access, one-click authentication, social logins, and notifications—streamlining their experience while enabling advanced features like on-ramp functionality, in-app token swaps and smart accounts.

> ### WalletKit

Reviewer: ```suggestion
## WalletKit
```
